spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/DirectReplyToMessageListenerContainer.java

```diff
@@ -82,33 +82,40 @@ public final boolean removeQueueNames(String... queueNames) {
 	}
 
 	@Override
+	@SuppressWarnings("deprecation")
```

Comment:
I somehow think that this and the next `setChannelAwareMessageListener` have to be marked as `@Deprecated` as well...